Comanche055/UPDATE_PROGRAM.agc

```diff
@@ -381,7 +381,7 @@ TIMEDIDL	EXTEND
 
 TIMEDIDR	INHINT
 		CAF	ZERO
-		ZL			# PICK UP INCRMENTER (AND ZERO
```

Comment:
No space between INCREMENTER and left parenthesis. ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1504.jpg))